docs/blog/newsletter-february-2021.rst

```diff
@@ -0,0 +1,88 @@
+
+So what's been happening on Slack?
+
+------------------------------------------
+Getting the right balance with screenshots
+------------------------------------------
+
+In December, documentarians had a lively discussion about how many screenshots to include in docs. As you might imagine, perspectives ranged from "absolutely no screenshots ever" to "as many screenshots as possible," but the most common opinion was "it depends" (surprise!). Most said they try to help both users who like a visual walkthrough _and_ users who prefer to follow text instructions. One writer approached the problem by presenting text explanations as the default, with an optional collapsible version that includes screenshots.
+
+Fewer screenshots means less maintenance work. If the product changes, the screenshots must change too, to remain helpful and prevent confusion. Lots of screenshots plus frequent product changes can cost a lot of time: keeping the docs in sync with the product can become unmanageable.
+
+A middle-ground approach is using text descriptions of UI elements, like "Click the START button", as it's easier to keep text descriptions matching the UI. And well-designed user interfaces and UI microcopy often mean that users don't need screenshots to find their way through the product.
+
+What about a video walkthrough or GIFs? Again, documentarians were split: it's helpful to watch a process through before trying it yourself, but videos and GIFs are inaccessible for people with visual impairments, distracting for people with attention issues, and unusable for people who don't have dependable broadband service. And following video instructions can be frustrating if you have to repeatedly press pause, or feel like the pace of spoken instructions is too slow.
+
+For more on screenshots in documentation, check out Steve Stegelin's `Graphic Content Warning: The Pros, Cons, and Alternatives to Screenshots <https://www.writethedocs.org/videos/portland/2018/graphic-content-warning-the-pros-cons-and-alternatives-to-screenshots-steve-stegelin/>`_ from Write the Docs Portland 2018 and Swapnil Ogale's `When bad screenshots happen to good writers <https://www.writethedocs.org/videos/eu/2016/when-bad-screenshots-happen-to-good-writers-swapnil-ogale/>`_ from WTD EU 2016.
+
+--------------------------------------
+Report from the book club: managing up
+--------------------------------------
+
+This winter, the `Write the Docs book club <https://app.slack.com/client/T0299N2DL/C7YJR1N02>`__ is reading and discussing their way through  Managing Writers by Richard Hamilton. The book covers a broad range of topics on the overall topic of managing docs projects and teams (both big and small). There’s been some fab discussion every week, so here’s a sample on the theme of showing your value and finding your place in the org.
```

Comment:
...topics on the overall topic...

Maybe "...topics on the overall theme..." instead?

Reply:
D'oh. Fixed.